client/ayon_shotgrid/plugins/publish/integrate_shotgrid_publish.py

```diff
@@ -141,3 +168,43 @@ def process(self, instance):
                 )
             instance.data["shotgridPublishedFile"] = sg_published_file
 
+    def _find_published_file_type(self, instance, filepath, representation):
```

Comment:
this function is a pretty simplistic way to set some types on the PublishedFiles of SG and it relies on setting these on your SG instance:
<img width="251" alt="Screenshot 2023-12-01 at 1 38 55 PM" src="https://github.com/ynput/ayon-shotgrid/assets/4348536/8c2b5055-7857-47f7-97d5-45423f125100">

Reply:
Yeah this is great too, how would you select what SH publish type (representation) you're publishing too?
Do I need to make sure I name the SG and AYON representations exactly the same?